src/panels/config/zha/zha-device-card.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  TemplateResult,
+  CSSResult,
+  css,
+} from "lit-element";
+import "@polymer/paper-item/paper-icon-item";
+import "@polymer/paper-item/paper-item-body";
+import "@polymer/paper-card/paper-card";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { haStyle } from "../../../resources/ha-style";
+import { HomeAssistant } from "../../../types";
+
+import "../../../components/entity/state-badge";
+import { ZHADevice } from "../../../data/zha";
+
+class ZHADeviceCard extends LitElement {
+  public hass?: HomeAssistant;
+  public narrow?: boolean;
+  private device?: ZHADevice;
+
+  constructor() {
+    super();
+  }
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      narrow: {},
+      device: {},
+    };
+  }
+
+  protected render(): TemplateResult | void {
+    return html`
+      <paper-card>
+        <div class="card-content">
+          <dl>
+            <dt class="label">IEEE:</dt>
+            <dd class="info">${this.device!.ieee}</dd>
+            <dt class="label">Quirk applied:</dt>
+            <dd class="info">${this.device!.quirk_applied}</dd>
+            <dt class="label">Quirk:</dt>
+            <dd class="info">${this.device!.quirk_class}</dd>
+          </dl>
+        </div>
+
+        <div class="device-entities">
+          ${this.device!.entities.map(
```

Comment:
Can you add a message if there are 0 entities? Or is that impossible?

Reply:
every device will have at least one